styleguide/components/41-organisms/timeline/_timeline.scss

```diff
@@ -0,0 +1,397 @@
+dl.timeline {
+  width: 100%;
+
+  * {
+    font-size: .8rem;
+   }
+
+  .timeline-slot {
+    @include theme('background-color', 'color-secondary', 'timeline-slot-default-background');
+
+    display: flex;
+    position: relative;
+    flex-wrap: wrap;
+    width: 100%;
+
+    // Alternating timeline-slots get another background color.
+    &:nth-child(2n) {
+      @include theme('background-color', 'color-primary--lighten-4', 'timeline-slot-alternating-background');
+
+      // Fix for video on alternating timeline-slots.
+      > dd .video-wrapper::after {
+        @include theme('background-color', 'color-primary--lighten-4', 'timeline-slot-alternating-background');
+      }
+    }
+
+    // First timeline-slot specific styling.
+    &:first-of-type {
+      padding-top: 2rem;
+
+      > dt {
+        padding-top: 0;
+
+        &::before {
+          display: none;
+        }
+      }
+
+      // First definition description inside the first timeline-slot.
+      > dd:first-of-type {
+        margin-top: .3rem;
+        padding-top: 0;
+
+        // Change the definition description indicator position.
+        &::before {
+          top: 0;
+        }
+
+        // Fix whitespace.
+        .timeline-slot-header {
+          margin-top: -.3rem;
+        }
+      }
+    }
+
+    // Last timeline-slot last definition description specific styling.
+    &:last-of-type > dd:last-of-type {
+      border: 0;
+
+      // Change the definition description indicator position.
+      &::before {
+        left: -.4rem;
+      }
+
+      // Fix whitespace issue with the timeline border.
+      &::after {
+        @include theme('border-color', 'color-primary', 'timeline-line-color');
+
+        position: absolute;
+        top: 0;
+        left: 0;
+        height: 1.3rem;
+        border-left: 3px solid;
+        content: '';
+
+        @include tablet {
+          height: 2.3rem;
+        }
+      }
+
+      // Hide video dots on last timeline-slot's last definition description.
+      .responsive-video::before,
+      .audio-description::after {
+        display: none;
+      }
+    }
+
+    // Timeline-slot title styling.
+    &-title {
+      &[aria-expanded="true"] {
+        @include icon('minus');
+      }
+
+      &[aria-expanded="false"] {
+        @include icon('plus');
+      }
+
+      position: relative;
+      padding: 0;
+      border: 0;
+      background: transparent;
+
+      h3 {
+        width: 100%;
+        margin-bottom: .2rem;
+        font-size: .9rem;
+        text-align: left;
+      }
+
+      &::before {
+        position: absolute;
+        left: -1.6rem;
+        font-size: 1.6em;
+      }
+    }
+
+    &-date {
+      display: block;
+      width: 100%;
```

Comment:
obsolete, display block is 100%

Reply:
This is necessary or the title and date div / span will be next to each other...

Reply:
No, that was caused by the template error.